src/stream.c

```diff
@@ -1611,7 +1611,8 @@ int stream_open_natpinhole(struct stream *strm)
 	if (!strm)
 		return EINVAL;
 
-	if (strm->pinhole)
+	if (strm->pinhole && (!strm->mnat ||
+			      !str_cmp(strm->mnat->id, "stun")))
```

Comment:
not sure about this. This means that the core gets knowledge about module-names.

I would prefer to delete the STUN module instead, and force ICE usage.

Reply:
Ah, okay. Is ICE without a TURN server possible?
Does this open the NAT pinhole?